### ISSUE TITLE
RAW_MQ/sumSquaredMQs parsing error when running GenotypeGVCFs for JointGenotyping

### ISSUE THREAD

**Issue by indraniel** [broadinstitute/gatk#5433]
This issue affect version of GATK4 from version 4.0.10.1 and onwards, and is related to github feature/pull request [#4969: "Improve MQ calculation accuracy"](https://github.com/broadinstitute/gatk/pull/4969).

GVCFs with large `RAW_MQ` sum of sumSquaredMQs values end up with an error like:

```
A USER ERROR has occurred: Bad input: malformed RAW_MQ annotation: 3415207168,1749038
```

The error happens when the sumSquaredMQs element value is greater than Java's. `Integer.MAX_VALUE`.  See the following [github pull request comment for more details](https://github.com/broadinstitute/gatk/pull/4969#issuecomment-439642813).  

Should a `java.lang.Long`, `java.math.BigInteger`, `java.lang.Double`, or `java.math.BigDecimal` be used here instead of a `java.lang.Integer` for method: [`parseRawDataString` inside `org.broadinstitute.hellbender.tools.walkers.annotator.RMSMappingQuality`](https://github.com/broadinstitute/gatk/blob/master/src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/RMSMappingQuality.java#L251) ?

I've noticed this error when using GATK4 version 4.0.11.0 on the [GATK4 Germline SNPs-INDELs WDL workflow](https://github.com/gatk-workflows/gatk4-germline-snps-indels).  It happens on the [`JointGenotyping.GenotypeGVCFs` task](https://github.com/gatk-workflows/gatk4-germline-snps-indels/blob/master/joint-discovery-gatk4-local.wdl#L127).

**Comment by indraniel** [comment]
Looks like the work in [pull request 5435](https://github.com/broadinstitute/gatk/pull/5435) is related to this issue.

**Comment by TedBrookings** [comment]
Yes, I messed up the commit message and wrote that it "resolves issue 5433" instead of "resolves issue #5433". Without the hash I guess github assumes it's stupid meatbag language and ignores it. The bug should be fixed so I'm closing the issue manually. Sorry to leave this hanging the last month, and thanks for pointing it out.

**Comment by indraniel** [comment]
No worries!  Thanks again for the fix!
